Expensify/react-native-wallet
# Set up the repo structure

### Details

This PR sets up the repo structure for the upcoming `react-native-wallet` library. It has been configured similarly to the [react-native-live-markdown](https://github.com/Expensify/react-native-live-markdown) and supports Fabric and Paper architectures. It also uses `npm` instead of `yarn` to make it consistent with other Expensify-related projects

![Screenshot 2025-01-08 at 12 42 32](https://github.com/user-attachments/assets/9647d750-4e99-499e-9825-35974d6c5c3f)


### Related Issues

https://github.com/Expensify/App/issues/36957

### Manual Tests

1. Clone the repo
2. In the main folder run `npm i`
3. Build the example app on both iOS and Android devices, using following architectures:
	- Fabric: just run `npm start` in the example folder
	- Paper:
		- iOS: Change `ENV['RCT_NEW_ARCH_ENABLED']` variable to `0` in *Podfile*
		- Android: Change the `newArchEnabled` variable to `false` in *gradle.properties*
4. Verify if all example apps are built successfully

### Linked PRs
